lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -115,5 +115,4 @@
     JsRunSerialized
     JsCreatePropertyId
     JsCopyPropertyId
```

Comment:
@obastemur If that's the case, that indicates "JsDiagEvaluate" and all other APIs there are unavailable in static lib?